tools/ci_build/build.py

```diff
@@ -331,6 +338,9 @@ def generate_build_tree(cmake_path, source_dir, build_dir, cuda_home, cudnn_home
     if is_windows():
         cmake_args += cmake_extra_args
 
+    if is_linux() and is_x64() and args.x86:
```

Comment:
shouldn't this if condition be triggered for either x86_64 OR AMD64 since technically in both cases, it would be treated as cross-compiling ?

Reply:
Adding AMD64 in next commit.